### ISSUE TITLE
After LFG bug.

### ISSUE THREAD

**Issue by DeadlyDraft** [Atlantiss/BugTracker#6890]
Hi.
For start, my english is not good but I hope you will understand me :P There is big bug after leaving LFG. My all stats just increase to really big value. I've got 80lvl and this bug just change for example my health points from 26k to 148k. The same with damage (300k crit from one skill) and other stats. I killed Gold Elite (80lvl) in maybe 2 seconds. It's not normal :P It didn't happen all the time when I left LFG. But when it happened, only join to LFG again fixed this issue. Screen Shoots should show everything.
Regards :)
![wowscrnshot_031816_214826](https://cloud.githubusercontent.com/assets/17936459/13893393/3fecb252-ed5e-11e5-9daf-4e1023bd1e88.jpg)
![wowscrnshot_031816_214835](https://cloud.githubusercontent.com/assets/17936459/13893397/454f0e0c-ed5e-11e5-85ca-03d002a65dbf.jpg)
![wowscrnshot_031816_215002](https://cloud.githubusercontent.com/assets/17936459/13893399/49201116-ed5e-11e5-8815-1640b0f316fc.jpg)
![wowscrnshot_031816_215604](https://cloud.githubusercontent.com/assets/17936459/13893401/4b8469ac-ed5e-11e5-9568-dfbba0497904.jpg)
![wowscrnshot_031816_215613](https://cloud.githubusercontent.com/assets/17936459/13893403/4e904f6c-ed5e-11e5-9a04-64377518ca3c.jpg)
![wowscrnshot_031816_220105](https://cloud.githubusercontent.com/assets/17936459/13893405/50dcbe54-ed5e-11e5-8f69-909d1b25d372.jpg)
![wowscrnshot_031816_220124](https://cloud.githubusercontent.com/assets/17936459/13893406/52d78f90-ed5e-11e5-81bb-84765479f3c7.jpg)
![wowscrnshot_031816_220135](https://cloud.githubusercontent.com/assets/17936459/13893409/54fb2f70-ed5e-11e5-866c-c40bc5137d07.jpg)

**Comment by Kiperr** [comment]
after which dungeon it happends?

**Comment by DeadlyDraft** [comment]
Yes. We've all died and we left. When I joined to dungeon I was fighting with mob and when I left dungeon I was ghost. After this I went to kill next mob and I kill him with one hit. This was weird and then I saw that all my stats have bugged :P

**Comment by DeadlyDraft** [comment]
I've find it. Propably it was Throne of the Tides.

**Comment by Thefakemccoy** [comment]
Sounds like the buff the last boss gives you at the last phase.

**Comment by DeadlyDraft** [comment]
Ok, but this buff shouldn't work if I leave dungeon. Even 85lvl couldn't kill me with this.

**Comment by Thefakemccoy** [comment]
The buff is tidal surge
http://www.wowhead.com/spell=76133/tidal-surge
